RecoDataProducts/inc/BkgCluster.hh

```diff
@@ -17,13 +17,13 @@ namespace mu2e
    {
        //Default hit count chosen for compuational efficiency
        BkgCluster() {_hits.reserve(16);}
-       BkgCluster(TwoDPoint point, float time) : _point(point), _time(time) {_hits.reserve(16);_cpoints.addPoint(_point,0);}
+       BkgCluster(TwoDPoint point, float time) : _point(point), _time(time) {_hits.reserve(16);_cpoints.addPoint(_point,0);_pos = point.pos3();_pos = point.pos3();}
```

Comment:
_pos is set twice.  It can also be set on initialization.

Reply:
Ooops, didn't see that in time.